package.json

```diff
@@ -8,7 +8,8 @@
     "build": "tsc -b && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "test": "jest"
+    "test": "jest",
+    "babel": "babel --help"
```

Comment:
You can remove the "babel" entry + the comma after jest

Reply:
removed